File: Documentation/3.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/Transformers/HowTo/ChangeTransformerLockMode.cs
using Raven.Abstractions.Indexing;
using Raven.Client.Document;

namespace Raven.Documentation.Samples.ClientApi.Commands.Transformers.HowTo
{
	public class ChangeTransformerLockMode
	{
		private interface IFoo
		{
			#region change_transformer_lock_1
			void SetTransformerLock(string name, TransformerLockMode lockMode);
			#endregion
		}

		public ChangeTransformerLockMode()
		{
			using (var store = new DocumentStore())
			{
				#region change_transformer_lock_2
				store.DatabaseCommands.SetTransformerLock("Orders/Company", TransformerLockMode.LockedIgnore);
				#endregion
			}
		}
	}
}

File: Documentation/3.5/Samples/java/src/test/java/net/ravendb/clientapi/commands/transformers/howto/ChangeTransformerLockMode.java
package net.ravendb.clientapi.commands.transformers.howto;

import net.ravendb.abstractions.indexing.TransformerLockMode;
import net.ravendb.client.IDocumentStore;
import net.ravendb.client.document.DocumentStore;

public class ChangeTransformerLockMode {

    private interface IFoo {
        //region change_transformer_lock_1
        public void setTransformerLock(String name, TransformerLockMode lockMode);
        //endregion
    }

    public ChangeTransformerLockMode() {
        try (IDocumentStore store = new DocumentStore()) {
            //region change_transformer_lock_2
            store.getDatabaseCommands().setTransformerLock("Orders/Company", TransformerLockMode.LOCKED_IGNORE);
            //endregion
        }
    }
}
